staking/programs/integrity-pool/src/lib.rs

```diff
@@ -68,6 +70,7 @@ pub mod integrity_pool {
             owner: payer.to_account_info(),
             target_account: None,
             pool_authority: Some(pool_config.to_account_info()),
+            system_program: ctx.accounts.system_program.to_account_info(),
```

Comment:
let's keep it consistent to define variables for accounts at the top of the function.

Reply:
done